internal/compiler/emitter.go

```diff
@@ -128,6 +129,11 @@ func getScriptTransformers(emitContext *printer.EmitContext, host printer.EmitHo
 
 	// transform module syntax
 	tx = append(tx, getModuleTransformer(&opts))
+
+	// inlining (formerly done via substitutions)
+	if !options.IsolatedModules.IsTrue() {
+		tx = append(tx, inliners.NewConstEnumInliningTransformer(&opts))
+	}
```

Comment:
Is not inlining when isolatedModules is enabled a new behavior in Corsa?

Reply:
No - I just lifted the check from inside the `ts` transform out to transform composition time.

Reply:
TIL.